RELEASE_NOTES.md

```diff
@@ -2,6 +2,10 @@
 
 ## Next release
 
+### Bug fixes
+
+- Fix comparing neighbor's timers pointers instead of their values.
```

Comment:
can you put the PR # and a more user friendly description of the bug (i.e. merging always fails if holdtime / keepalive time is set)?

Reply:
done